appservice/src/createAppService/IAppServiceWizardContext.ts

```diff
@@ -61,8 +61,15 @@ export interface IAppServiceWizardContext extends IResourceGroupWizardContext, I
 
     /**
      * The runtime to put to the top of the QuickPick list to recommend to the user.
-     * This should be set in `setWizardContextDefaults`
+     * This should be set in `getWizardRecommendations`
      */
 
-     detectedSiteRuntime?: string;
+    recommendedSiteRuntime?: LinuxRuntimes;
+
+    /**
+     * The OS that we detect that the user should use.  Will be set automatically for Basic Creation.
+     * This should be set in `getWizardRecommendations`
+     */
+
+    recommendedWebsiteOS?: WebsiteOS;
```

Comment:
This has the same naming as `recommendedSiteRuntime` but it's used in a different way (aka you don't use it to sort the quick pick). It's not even used in the wizard, which is the whole point of the "WizardContext"